ci/docker/runtime_functions.sh

```diff
@@ -637,9 +685,11 @@ build_ubuntu_gpu_cmake_mkldnn() {
 build_ubuntu_gpu_cmake() {
     set -ex
     cd /work/build
+    build_ccache_wrappers
```

Comment:
Why we need both?

Reply:
Because NVCC in cmake below 3.10 doesn't redirect the calls to g++ to ccache. Thus, we have to do both. Also, submodules - again :(

Reply:
@larroy @marcoabreu based on the update to cmake 3.13, now `build_ccache_wrappers` is only needed due to some submodules not handling the `CMAKE_CXX_COMPILER_LAUNCHER` correctly?

Reply:
those submodules should use CC and CXX variables, otherwise they are broken. We would need to check.

Reply:
I'd recommend to verify. Some submodules used to ignore the standards. Cmake as well as make paths should be tested - sometimes, cmake and make get intermingled